pypika/terms.py

```diff
@@ -231,6 +233,18 @@ def _get_value_sql(self):
         return str(self.value)
 
 
+class Values(Term):
+    def __init__(self, field,):
+        super(Values, self).__init__(None)
+        self.field = Field(field) if not isinstance(field, Field) else field
+
+    def fields(self):
+        return []
+
+    def get_sql(self, quote_char=None, **kwargs):
+        return f'VALUES({self.field.get_sql(quote_char=quote_char, **kwargs)})'
```

Comment:
Can't use a format string here, this needs to support python 2.7 and 3+

Reply:
true thanks. and already noticed thanks to the build 👍